app/src/main/java/org/sagebionetworks/bridge/researchstack/CrfDataProvider.java

```diff
@@ -50,9 +50,12 @@
 
     public static final String CLINIC1 = "clinic1";
     public static final String CLINIC2 = "clinic2";
+    public static final String TEST_USER = "test_user";
+    public static final String UX_TESTER = "ux_tester";
+
+    public static final Set<String> HIDDEN_TASK_IDS = ImmutableSet.of(CLINIC1, CLINIC2);
+    public static final Set<String> TEST_DATA_GROUPS = ImmutableSet.of(TEST_USER, UX_TESTER);
```

Comment:
Including both of these in TEST_DATA_GROUPS is a little misleading.  TEST_USER is the only data group that affects anything in the app the user will see, UX_TESTER only applies to tagging uploads as unimportant.  Maybe some comments to clarify?